### PR TITLE
Fix misc. grammar and minor details in osu!taiko article

### PR DESCRIPTION
**Pull request by SherlockWh0**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)

### CODE REVIEW

File: wiki/Game_mode/osu!taiko/en.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ osu!taiko is a game mode in osu! based on the Japanese rhythm game series, [Taik
 
 ### Song Selection
 
-To access the osu!taiko game mode, press `Ctrl` + `2` at the same time.
+To access the osu!taiko game mode, press `Ctrl` &  `2` at the same time.
```
Comment (iSlodinxOsu) [inline wiki/Game_mode/osu!taiko/en.md:15]: Is this really necessary?

Comment (SherlockWh0) [reply wiki/Game_mode/osu!taiko/en.md:15]: in my opinion it flows better

Comment (iSlodinxOsu) [reply wiki/Game_mode/osu!taiko/en.md:15]: most of the cases, mostly its mentioned with `+` than `&`

Comment (peppy) [reply wiki/Game_mode/osu!taiko/en.md:15]: using `&` here is indeed wrong.

Comment (SherlockWh0) [reply wiki/Game_mode/osu!taiko/en.md:15]: will change

Comment (SherlockWh0) [reply wiki/Game_mode/osu!taiko/en.md]: done

File: wiki/Game_mode/osu!taiko/en.md [hunk 2]
```diff
@@ -12,7 +12,7 @@ osu!taiko is a game mode in osu! based on the Japanese rhythm game series, [Taik
 
 ### Song Selection
 
-To access the osu!taiko game mode, press `Ctrl` + `2` at the same time.
+To access the osu!taiko game mode, press `Ctrl` +  `2` at the same time.
```
Comment (iSlodinxOsu) [inline wiki/Game_mode/osu!taiko/en.md:15]:
```suggestion
To access the osu!taiko game mode, press `Ctrl` + `2` at the same time.
```

**Review by iSlodinxOsu — CHANGES_REQUESTED** [review]
Remove space

**Review by Hiviexd — DISMISSED** [review]

File: wiki/Game_mode/osu!taiko/en.md [hunk 3]
```diff
@@ -27,14 +27,14 @@ For players with prior *Taiko no Tatsujin* experience:
 - No chibi dancers at the bottom (must be storyboarded)
 - The health bar must be at least 50% full to pass the song
 - *Kiai Time* triggers the *"Go-Go Time"*
-  - Gameplay gimmicks like the hit balloons or forked paths are not implemented (only the barebone basics)
+  - Gameplay gimmicks like the hit balloons or forked paths are not implemented (only the basics)
 - Background difference
   - A video or static background image-only beatmap will take up the lower portion of the screen
-  - If the map has been storyboarded, the storyboard will take over the bottom of the screen
+  - If the map has been storyboarded, the storyboard will override the bottom of the screen
```
[the remark this answers is not in the document]
Comment (SherlockWh0) [reply wiki/Game_mode/osu!taiko/en.md]: done

File: wiki/Game_mode/osu!taiko/en.md [hunk 4]
```diff
@@ -27,14 +27,14 @@ For players with prior *Taiko no Tatsujin* experience:
 - No chibi dancers at the bottom (must be storyboarded)
 - The health bar must be at least 50% full to pass the song
 - *Kiai Time* triggers the *"Go-Go Time"*
-  - Gameplay gimmicks like the hit balloons or forked paths are not implemented (only the barebone basics)
+  - Gameplay gimmicks like the hit balloons or forked paths are not implemented (only the basics)
 - Background difference
   - A video or static background image-only beatmap will take up the lower portion of the screen
-  - If the map has been storyboarded, the storyboard will take over the bottom of the screen
+  - If the map has been storyboarded, the storyboard will stay a layer behind the bottom of the screen
```
Comment (Hiviexd) [inline wiki/Game_mode/osu!taiko/en.md:33]:
```suggestion
  - If the map has been storyboarded, the storyboard will take over the entire screen, but stay a layer behind the playfield
```

**Review by cl8n — APPROVED** [review]
not that it isn't an improvement, but I don't think it's really worth the time to worry about small details like this in an article that at large is very low quality and will likely need a full rewrite before that `needs_cleanup` label is removed.

**Pull request merged**